doc/clang.txt

```diff
@@ -227,6 +227,16 @@ g:clang_use_path                        *g:clang_use_path*
     Default: >
         let g:clang_use_path = 1
 <
+g:clang_verbose_menu                        *g:clang_verbose_menu*
```

Comment:
Use `g:clang_verbose_pmenu`  ? `pmeun` is short for popup menu, the official name.